diff --git a/src/schema.rs b/src/schema.rs
index af79bf5..4dfba0a 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -4,7 +4,8 @@
         queue_id -> Nullable<Int4>,
         exchange_id -> Nullable<Int4>,
         routing_key -> Nullable<Varchar>,
-        _nowait -> Nullable<Bool>,
+        #[sql_name = "_nowait"]
+        nowait -> Nullable<Bool>,
         arguments -> Nullable<Jsonb>,
     }
 }
@@ -12,11 +13,13 @@
 table! {
     exchange (id) {
         id -> Int4,
-        _name -> Nullable<Varchar>,
+        #[sql_name = "_name"]
+        name -> Nullable<Varchar>,
         passive -> Nullable<Bool>,
         durable -> Nullable<Bool>,
         auto_delete -> Nullable<Bool>,
-        _nowait -> Nullable<Bool>,
+        #[sql_name = "_nowait"]
+        nowait -> Nullable<Bool>,
         arguments -> Nullable<Jsonb>,
     }
 }
@@ -36,12 +39,15 @@
 table! {
     queue (id) {
         id -> Int4,
-        _name -> Nullable<Varchar>,
+        #[sql_name = "_name"]
+        name -> Nullable<Varchar>,
         passive -> Nullable<Bool>,
         durable -> Nullable<Bool>,
-        _exclusive -> Nullable<Bool>,
+        #[sql_name = "_exclusive"]
+        exclusive -> Nullable<Bool>,
         auto_delete -> Nullable<Bool>,
-        _nowait -> Nullable<Bool>,
+        #[sql_name = "_nowait"]
+        nowait -> Nullable<Bool>,
         arguments -> Nullable<Jsonb>,
     }
 }
@@ -50,9 +56,4 @@
 joinable!(bind -> queue (queue_id));
 joinable!(message -> queue (queue_id));
 
-allow_tables_to_appear_in_same_query!(
-    bind,
-    exchange,
-    message,
-    queue,
-);
+allow_tables_to_appear_in_same_query!(bind, exchange, message, queue);
