diff --git a/src/schema.rs b/src/schema.rs
index af79bf5..4dfba0a 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,25 +1,28 @@
 table! {
     bind (id) {
         id -> Int4,
         queue_id -> Nullable<Int4>,
         exchange_id -> Nullable<Int4>,
         routing_key -> Nullable<Varchar>,
-        _nowait -> Nullable<Bool>,
+        #[sql_name = "_nowait"]
+        nowait -> Nullable<Bool>,
         arguments -> Nullable<Jsonb>,
     }
 }
 
 table! {
     exchange (id) {
         id -> Int4,
-        _name -> Nullable<Varchar>,
+        #[sql_name = "_name"]
+        name -> Nullable<Varchar>,
         passive -> Nullable<Bool>,
         durable -> Nullable<Bool>,
         auto_delete -> Nullable<Bool>,
-        _nowait -> Nullable<Bool>,
+        #[sql_name = "_nowait"]
+        nowait -> Nullable<Bool>,
         arguments -> Nullable<Jsonb>,
     }
 }
 
 table! {
     message (id) {
@@ -33,18 +36,21 @@ table! {
     }
 }
 
 table! {
     queue (id) {
         id -> Int4,
-        _name -> Nullable<Varchar>,
+        #[sql_name = "_name"]
+        name -> Nullable<Varchar>,
         passive -> Nullable<Bool>,
         durable -> Nullable<Bool>,
-        _exclusive -> Nullable<Bool>,
+        #[sql_name = "_exclusive"]
+        exclusive -> Nullable<Bool>,
         auto_delete -> Nullable<Bool>,
-        _nowait -> Nullable<Bool>,
+        #[sql_name = "_nowait"]
+        nowait -> Nullable<Bool>,
         arguments -> Nullable<Jsonb>,
     }
 }
 
 joinable!(bind -> exchange (exchange_id));
 joinable!(bind -> queue (queue_id));
